doc/user/layouts/shortcodes/postgres-direct/create-a-publication-aws.html

```diff
@@ -46,6 +46,10 @@
     additional tables, Materialize will waste resources on ingesting and then
     immediately discarding the data.
 
+    Additionally, make sure you do not include any tables you did not set to 
```

Comment:
I made this mistake and it was annoying to unwork.  That said, I'm not the target audience so maybe others wouldn't.

Reply:
I ran into this issue because I was trying to save time and not list out every individual table so I did a CREATE SOURCE FOR ALL in postgres and then was blocked in Materialize.  That said I'm not really the target here so a non-lazy person trying may not run into this